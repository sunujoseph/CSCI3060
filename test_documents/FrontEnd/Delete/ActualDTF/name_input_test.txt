02 userBS AA 000000000
