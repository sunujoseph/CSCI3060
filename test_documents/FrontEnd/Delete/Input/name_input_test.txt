login
admin1
delete
userBS
