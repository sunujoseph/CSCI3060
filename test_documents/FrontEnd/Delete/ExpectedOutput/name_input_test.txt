Enter Username: 
Enter a user name to delete the account of: 
User: userBS, has been deleted.
